src/main/java/com/google/codeu/data/Message.java

```diff
@@ -30,15 +32,18 @@
    * Constructs a new {@link Message} posted by {@code user} with {@code text} content. Generates a
    * random ID and uses the current system time for the creation time.
    */
-  public Message(String user, String text) {
-    this(UUID.randomUUID(), user, text, System.currentTimeMillis());
+  //add recipient to the constructor
+  public Message(String user, String text, String recipient) {
+    this(UUID.randomUUID(), user, text, System.currentTimeMillis(), recipient);
   }
 
-  public Message(UUID id, String user, String text, long timestamp) {
+  //add recipient to the constructor
+  public Message(UUID id, String user, String text, long timestamp, String recipient) {
```

Comment:
Javadoc comments are in the style:

/**
/* Constructs a new message.
/* 
/* @param id A non-null uuid to uniquely identify the constructed message.
...

Reply:
I've added comment for every method.